castanet/counter.py

```diff
@@ -121,8 +135,16 @@ def sum_cast_dict(cast_dict):
     return total
 
 def assignment_count(cast_dict):
-    """A function for counting the number of assignmnet"""
-    # An example of an assignmnet is x = y
+    """A function for counting the number of assignmnet
+
+    Args:
+        cast_dict: adds the number of assignment to a dictionary
```

Comment:
This should read `cast_dict: A dictionary of files and the corresponding CAST's`.